GeosCore/kpp_standalone_interface.F90

```diff
@@ -0,0 +1,698 @@
+      USE TIME_MOD,                 ONLY : GET_TS_CHEM
+      USE TIME_MOD,                 ONLY : TIMESTAMP_STRING
+      USE TIME_MOD,                 ONLY : Get_Minute
+      USE TIME_MOD,                 ONLY : Get_Hour
+      USE TIME_MOD,                 ONLY : Get_Day
+      USE TIME_MOD,                 ONLY : Get_Month
+      USE TIME_MOD,                 ONLY : Get_Year
+      USE Pressure_Mod,             ONLY : Get_Pcenter
+      USE inquireMod,               ONLY : findFreeLUN
+! !INPUT PARAMETERS:
+!
+      INTEGER,        INTENT(IN)    :: I            ! Longitude index
+      INTEGER,        INTENT(IN)    :: J            ! Latitude index
+      INTEGER,        INTENT(IN)    :: L            ! GEOS-Chem vertical level
+      INTEGER,        INTENT(IN)    :: KPP_TotSteps ! Total KPP integrator steps
+
+      TYPE(GrdState), INTENT(IN)    :: State_Grid                 ! Grid State object
+      TYPE(ChmState), INTENT(IN)    :: State_Chm                  ! Chemistry State object
+      TYPE(MetState), INTENT(IN)    :: State_Met                  ! Meteorology State object
+      TYPE(OptInput), INTENT(IN)    :: Input_Opt                  ! Input Options object
+      REAL(dp), INTENT(IN)          :: initC(NSPEC)               ! Initial concentrations
+      REAL(dp), INTENT(IN)          :: localRCONST(NREACT)        ! Rate constants
+      REAL(dp)                      :: initHvalue                 ! Initial timestep
+
+! !OPTIONAL INPUT PARAMETER
+      LOGICAL, OPTIONAL             :: FORCE_WRITE  ! Write even if not in an active cell
+      CHARACTER(LEN=*), OPTIONAL    :: CELL_NAME    ! Customize the name of this file 
+!
+! !AUXILLIARY LOCAL PARAMETERS (pass the aux bc Fortran doesn't have defaults for kwargs)
+      LOGICAL            :: FORCE_WRITE_AUX   ! Write even if not in an active cell
+      CHARACTER(LEN=255) :: CELL_NAME_AUX     ! Customize the name of this file 
+!
+! !OUTPUT PARAMETERS:
+!
+      INTEGER,        INTENT(OUT)   :: RC          ! Success or failure
+!
+! !LOCAL VARIABLES:
+      ! Integers
+      INTEGER                :: N          ! Loop index
+      INTEGER                :: IU_FILE    ! Available unit for writing
+      INTEGER                :: SpcID      ! Mapping from State_Chm and KPP
+      REAL(fp)               :: DT         ! Chemistry operator timestep
+      ! Strings
+      CHARACTER(LEN=255)     :: YYYYMMDD_hhmmz
+      CHARACTER(LEN=255)     :: level_string
+      CHARACTER(LEN=512)     :: errMsg
+      
+      ! Arrays
+      REAL(dp)               :: Vloc(NVAR), Aout(NREACT)  ! For KPP reaction rate diagnostics
+
+
+      ! Did a user want to write the chemical state even if not in an active cell?
+      IF ( PRESENT(FORCE_WRITE) ) THEN
+         FORCE_WRITE_AUX = FORCE_WRITE
+      ELSE
+         FORCE_WRITE_AUX = .FALSE.
+      END IF
+
+      ! Quit early if there's no writing to be done
+      IF (KPP_Standalone_YAML%Active_Cell .eq. .FALSE. .AND. FORCE_WRITE_AUX .eq. .FALSE.) THEN 
```

Comment:
GNU fortran 10.2 is tripping up on this line. Better to use:
`IF ( (.not. KPP_Standalone_YAML%Active_Cell) .AND. (.not. FORCE_WRITE_AUX) ) THEN`

Reply:
Thanks! I replaced that line with your suggestion.